docs/rst/use_cases/keys/keys.rst

```diff
@@ -0,0 +1,171 @@
+* :ref:`linux_source_installation`
+* :ref:`docker_installation`
+
+
+Keyed Messages
+--------------
+
+The keyed Topics use the following message types to represent the turtles' velocity and pose:
+
+.. raw:: html
+
+    <div style="display: flex;">
+        <div style="flex: 1; padding: 10px;">
+            <p><b>docs_turtlesim::msg::KeyedPose</b></p>
+            <p>Represents the pose of a turtle with a key.</p>
+            <ul>
+                <li>key: Identifier of the turtle.</li>
+                <li>x: X-coordinate of the turtle's position.</li>
+                <li>y: Y-coordinate of the turtle's position.</li>
+                <li>theta: Orientation angle of the turtle.</li>
+            </ul>
+        </div>
+        <div style="flex: 1; padding: 10px;">
+            <img src="../../../_static/resources/use_cases/keys/keyed_pose.png" alt="KeyedPose" style="width: 220px;">
+        </div>
+    </div>
+
+.. raw:: html
+
+    <div style="display: flex;">
+        <div style="flex: 1; padding: 10px;">
+            <p><b>docs_turtlesim::msg::KeyedVelocity</b></p>
+            <p>Represents the velocity of a turtle with a key.</p>
+            <ul>
+                <li>key: Identifier of the turtle.</li>
+                <li>linear: Linear velocity of the turtle.</li>
+                <li>angular: Angular velocity of the turtle.</li>
+            </ul>
+        </div>
+        <div style="flex: 1; padding: 10px;">
+            <img src="../../../_static/resources/use_cases/keys/keyed_twist.png" alt="KeyedTwist" style="width: 320px;">
+        </div>
+    </div>
+
+
+Prepare the ROS 2 workspace
+---------------------------
+
+The ROS 2 package used in this use case is:
+
+* `docs_turtlesim <https://github.com/eProsima/vulcanexus/tree/main/code/turtlesim>`__: a simple ROS 2 package for teaching ROS 2 concepts.
+
+To create a new ROS 2 workspace and clone the `docs_turtlesim` package run:
+
+.. code-block:: bash
+
+    mkdir -p $HOME/ROS2-ws/src
+    cd $HOME/ROS2-ws/src
+    git clone --depth=1 https://github.com/eProsima/vulcanexus.git tmp_dir
+    cd tmp_dir
+    git archive --format zip --output ../turtlesim.zip HEAD:code/turtlesim
+    cd ..
+    unzip turtlesim.zip -d docs_turtlesim
+    rm -rf tmp_dir turtlesim.zip
+
+Build the ROS 2 workspace with:
+
+.. code-block:: bash
+
+    cd $HOME/ROS2-ws
+    colcon build
+
+The resulting directory structure should look like this:
+
+.. code-block:: bash
+
+    $HOME/ROS2-ws/
+    ├── build
+    ├── install
+    ├── log
+    └── src
+        └── docs_turtlesim
+
+Execution
+---------
+
+First, it is necessary to setup the Vulcanexus environment and the ROS 2 workspace in all the terminals.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    source $HOME/ROS2-ws/install/setup.bash
+
+Run Controller
+^^^^^^^^^^^^^^
+
+To start the turtlesim controller, run:
+
+.. code-block:: bash
+
+    ros2 run docs_turtlesim turtlesim_multi_teleop
+
+Run Qt application
```

Comment:
Same as before, this is just the turtlesim node but running 3 turtle nodes at once. That's on what you should focus on. Remove every Qt/GUI reference